### ISSUE TITLE
document how to change the debug state/level of an existing container

### ISSUE THREAD

**Issue by stuclem** [vmware/vic-product#510]
@chengwang86 commented on [Fri Apr 07 2017](https://github.com/vmware/vic/issues/4645)

**User Statement:**

As a VIC user, after I change the debug state of my VCH, new containers created by the VCH will take the updated debug state for logging. However, existing containers will still use the previous debug state. We need documentation on how to use `govc` or direct editing to update the debug state for such existing containers when needed. 

**Acceptance Criteria:**
- I should be able to update the debug state of an existing container following the instructions from the documentation.

**Comment by stuclem** [comment]
@chengwang86 I just found this one and migrated it to the vic-product repo. Do we need this in the doc for 1.2? Thanks!

**Comment by chengwang86** [comment]
@stuclem Yes, I think we do need this for 1.2 since the corresponding feature (vic-machine configure --debug) is in 1.2

**Comment by stuclem** [comment]
Awaiting confirmation from @chengwang86 as to whether this was implemented in 1.2.

**Comment by chengwang86** [comment]
@stuclem Here are the steps:

1. stop the container via `docker stop`
2. connect to the vcenter server via `govc`.
3.  update the debug level configuration for the container via `govc vm.change -vm vmName -e guestinfo.vice./diagnostics/debug=Debug_level`. Here `vmName` is usually `containerName-containerShortID` and can be replaced by `containerName*`. The user set the `Debug_level` to a desired integer.
4. start the container via `docker start`.

**Comment by stuclem** [comment]
Looks like this slipped through the net.

**Comment by zjs** [comment]
Perhaps this would be more appropriate for a KB (or iKB) as there's no user-friendly container debug logging.

**Comment by stuclem** [comment]
Moving this into the KB bucket.
